WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -627,18 +627,17 @@ private extension NotificationsViewController {
 //
 extension NotificationsViewController {
 
-    /// Called on view load to determine whether the Jetpack banner should be shown on the view
+    /// Called on viewWillAppear to determine whether the Jetpack banner should be shown on the view
     /// Also called in the completion block of the JetpackLoginViewController to show the banner once the user connects to a .com account
     func configureJetpackBanner() {
-        guard JetpackBrandingVisibility.all.enabled else {
-            return
-        }
-        jetpackBannerView.buttonAction = { [unowned self] in
-            JetpackBrandingCoordinator.presentOverlay(from: self)
-            JetpackBrandingAnalyticsHelper.trackJetpackPoweredBannerTapped(screen: .notifications)
+        if jetpackBannerView.buttonAction == nil {
```

Comment:
👋 @ipalm0423 - Could we keep the logic to determine whether or not to activate the banner local to this view?

We've followed this pattern for the other views, and although they currently all match the same logic, we aren't sure if some may differ in the future. Thanks!

Reply:
👋 @twstokes Ok, No problem. 
I change my fix at 7d39b158133a04205bc7e0d6876abc91e2a5cdef: 
This commit only impacts the `NotificationsViewController`, which makes the impaction smaller and easy to test.
After we know every specification on other views, we can open another PR to fix it.